addon/styles/_frost-subcontext-actions.scss

```diff
@@ -0,0 +1,85 @@
+$frost-subcontext-actions-button-bg: $frost-color-white;
+$frost-subcontext-actions-button-shadow: $frost-color-grey-6;
+$frost-subcontext-actions-button-icon: $frost-color-grey-4;
+$frost-subcontext-actions-button-icon-selected: $frost-color-blue-1;
+$frost-subcontext-actions-button-icon-active: $frost-color-blue-1;
+$frost-subcontext-actions-label-background: rgba(0, 0, 0, .65);
+@keyframes frost-subcontext-actions-label-animation-out {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+
+}
+
+@keyframes frost-subcontext-actions-label-animation-in {
+  to {
+    // right: 0%;
+    opacity: 0;
+    visibility: hidden;
+  }
+
+  from {
+    // right: 170%;
+    opacity: 1;
+    visibility: visible;
+  }
+
+}
+
+.frost-subcontext-actions {
+  width: 60px;
+  height: 100%;
+  margin-left: auto;
+  padding: 20px 15px;
+  border-left: 1px solid $frost-color-blue-1;
+  background-color: $frost-color-white;
+}
+
+.frost-subcontext-actions-button {
+  display: flex;
+  position: relative;
+
+  .frost-icon {
+    width: 30px;
+    height: 30px;
+    color: $frost-color-blue-1;
+  }
+
+  .frost-subcontext-actions-label {
+    display: none;
+    position: absolute;
+    right: 170%;
+    height: 35px;
+    max-height: 35px;
+    padding: 8px 15px 6px;
```

Comment:
Usually we do spacing in units of 5px - have @baconC take a look at the screenshot and approve these numbers